upstream_utils/upstream_utils.py

```diff
@@ -104,9 +104,9 @@ def copy_to(files, root):
 
         # Rename .cc file to .cpp
         if dest_file.endswith(".cc"):
-            dest_file = os.path.splitext(dest_file)[0] + ".cpp"
+            dest_file = f"{os.path.splitext(dest_file)[0]}.cpp"
         if dest_file.endswith(".c"):
-            dest_file = os.path.splitext(dest_file)[0] + ".cpp"
+            dest_file = f"{os.path.splitext(dest_file)[0]}.cpp"
```

Comment:
These two feel a bit harder to read.

Reply:
You are right. I usually try to use f-strings instead of regular ``+`` concatenations, but in this case, it makes the code more difficult to read.